README.md

```diff
@@ -1,5 +1,7 @@
 ![Run tests](https://github.com/alphagov/govuk-coronavirus-vulnerable-people-form/workflows/Run%20tests/badge.svg)
 
+This codebase is archived. It has been rebuilt [here](https://github.com/alphagov/govuk-shielded-vulnerable-people-service).
```

Comment:
```suggestion
This codebase is archived. It has been superseded by [alphagov/govuk-shielded-vulnerable-people-service](https://github.com/alphagov/govuk-shielded-vulnerable-people-service).
```

linking to "here" tends to be a bad practice: https://www.w3.org/QA/Tips/noClickHere

Reply:
Thanks @kevindew! I amended the commit.